src/libraries/System.Private.CoreLib/src/System/Convert.cs

```diff
@@ -2527,16 +2527,16 @@ private static unsafe int ConvertToBase64Array(char* outChars, byte* inData, int
 
         private static int ToBase64_CalculateAndValidateOutputLength(int inputLength, bool insertLineBreaks)
         {
-            long outlen = ((long)inputLength) / 3 * 4;          // the base length - we want integer division here.
-            outlen += ((inputLength % 3) != 0) ? 4 : 0;         // at most 4 more chars for the remainder
+            // the base length - we want integer division here, at most 4 more chars for the remainder
+            long outlen = ((long)inputLength + 2) / 3 * 4;
 
             if (outlen == 0)
                 return 0;
 
             if (insertLineBreaks)
             {
-                long newLines = outlen / base64LineBreakPosition;
-                if ((outlen % base64LineBreakPosition) == 0)
+                long newLines = Math.DivRem(outlen, base64LineBreakPosition, out long remainder);
```

Comment:
```suggestion
                (long newLines, long remainder) = Math.DivRem(outlen, base64LineBreakPosition);
```
We are using the new overload everywhere else in CoreLib.

Reply:
Does returning a tuple generate good code on all platforms now? I think I remember that this wasn't always the case.

Reply:
Yes, we have analyzed the codegen and converted number of DivRem use the tuple returning overload in https://github.com/dotnet/runtime/pull/45074

Reply:
Oh this is good to know